### DISCUSSION TITLE
Multi Tenant solutions with Blazor assembly

### DISCUSSION THREAD

**Discussion by abdtime** [ServiceStack/Discuss#45] [Ideas]
Hello,

I am looking for an advanced sample with Blazor.

I am intressed for pour product/project

Thanks

**Comment by mythz** [comment]
Thanks for the feedback, we have creating a Multi Tenant project solution on our TODO list, in the meantime please see our [Multitenancy docs](https://docs.servicestack.net/multitenancy) for how implement Multitenant APIs and how to implement them with [AutoQuery CRUD Attributes](https://docs.servicestack.net/autoquery-crud#autopopulate).
